modules/tide_site/src/AliasManager.php

```diff
@@ -26,9 +26,10 @@ class AliasManager extends CoreAliasManager {
   /**
    * {@inheritdoc}
    */
-  public function __construct(AliasRepositoryInterface $repository, AliasWhitelistInterface $whitelist, LanguageManagerInterface $language_manager, CacheBackendInterface $cache, protected ?TimeInterface $time = NULL, AliasStorageHelper $alias_helper) {
+  public function __construct(AliasRepositoryInterface $repository, AliasWhitelistInterface $whitelist, LanguageManagerInterface $language_manager, CacheBackendInterface $cache, protected ?TimeInterface $time = NULL)
+  {
     parent::__construct($repository, $whitelist, $language_manager, $cache, $time);
-    $this->aliasHelper = $alias_helper;
+    $this->aliasHelper = \Drupal::service('tide_site.alias_storage_helper');
```

Comment:
Using `$this->aliasHelper = \Drupal::service('tide_site.alias_storage_helper')` isn’t ideal because it bypasses dependency injection, making the code harder to test.

The original error, 
```bash
      8192: Optional parameter $time declared before required parameter $alias_helper is implicitly treated as a required parameter in docroot/modules/contrib/tide_core/modules/tide_site/src/AliasManager.php line 29 (Drupal\Core\Entity\EntityStorageException)
```

 happened because required parameters must come before optional ones. 
try to move `AliasStorageHelper $alias_helper` before $time

Reply:
> Using `$this->aliasHelper = \Drupal::service('tide_site.alias_storage_helper')` isn’t ideal because it bypasses dependency injection, making the code harder to test.
> 
> The original error,
> 
> ```shell
>       8192: Optional parameter $time declared before required parameter $alias_helper is implicitly treated as a required parameter in docroot/modules/contrib/tide_core/modules/tide_site/src/AliasManager.php line 29 (Drupal\Core\Entity\EntityStorageException)
> ```
> 
> happened because required parameters must come before optional ones. try to move `AliasStorageHelper $alias_helper` before $time

@vincent-gao , That was my first thought too. You can see in the first commit I have done what you are suggestions and it doesn't fixing the issue. https://github.com/dpc-sdp/tide_core/pull/531/commits/16c32f83f0b46ec18e09b7a71dd162b3977660ac

Reply:
😢